packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -2296,6 +2314,9 @@ msgstr "Nebyly nalezeny žádné slevové kupóny. Nejprve musíte nějaký vytv
 msgid "No records found."
 msgstr "Nebyly nalezeny žádné záznamy."
 
+msgid "No sales representatives found."
+msgstr "Doposud nebyly vytvořeny žádní obchodní zástupci."
```

Comment:
```suggestion
msgstr "Doposud nebyli vytvořeny žádní obchodní zástupci."
```

Reply:
should i change this for payment and transport messages as well?

Reply:
nope, for transport and payment it was correct (Czech grammar, I know :smile: )

Reply:
i give up :D now it is same as it was for these